### DISCUSSION TITLE
Provide support for PCRE2

### DISCUSSION THREAD

**Discussion by minhtrancccp** [jarun/nnn#1781] [Ideas]
As shown in [the instruction to make a PCRE-supported binary](https://github.com/jarun/nnn/wiki/Developer-guides#compile-with-pcre), the devel lib of older PCRE is required (which is not made clear from the instruction's text).
Since PCRE is no longer developed by its own maintainers from 2021, as well as a wider adoption for PCRE2 by other projects (such as PHP or grep), I think it would be appropriate for nnn's devs to migrate to PCRE2 instead, or at least to make a note on which PCRE version to be used in the make process.

**Comment by jarun** [comment]
Please raise a PR.

**Comment by jarun** [comment]
Added PCRE2 support at fe87551742b679f2adc2d912ba0ecf3bb9987a8e.
